chore(logging)_: switch to zap.Logger as central logger

osmaczko opened the pull request:
closes: https://github.com/status-im/status-go/issues/6029

NOTE: It will require testing on both platforms.

status-im-auto commented:
### Jenkins Builds
<details>
<summary>Click to see older builds (95)</summary>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_multiplication_x: | ec32ae57 | [#1](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6025/1/) | 2024-11-01 18:45:39 | ~4 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6025/1//consoleText) |
| :heavy_check_mark: | ec32ae57 | [#1](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6025/1/) | 2024-11-01 18:45:47 | ~4 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-241101-184119-ec32ae-pr6025.zip) |
| :heavy_check_mark: | ec32ae57 | [#1](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6025/1/) | 2024-11-01 18:45:49 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-241101-184114-ec32ae-pr6025.zip) |
| :heavy_check_mark: | ec32ae57 | [#1](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6025/1/) | 2024-11-01 18:46:09 | ~4 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6025/1//consoleText) |
| :heavy_check_mark: | ec32ae57 | [#1](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6025/1/) | 2024-11-01 18:46:24 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-241101-184111-ec32ae-pr6025.zip) |
| :heavy_check_mark: | ec32ae57 | [#1](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6025/1/) | 2024-11-01 18:46:43 | ~5 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-241101-184109-ec32ae-pr6025.aar) |
| :heavy_check_mark: | ec32ae57 | [#1](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6025/1/) | 2024-11-01 18:46:50 | ~5 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-241101-184110-ec32ae-pr6025.zip) |
| :heavy_check_mark: | ec32ae57 | [#1](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6025/1/) | 2024-11-01 18:47:36 | ~6 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-241101-184114-ec32ae-pr6025.zip) |
| | | | | | | |
| :heavy_check_mark: | aadc36df | [#2](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6025/2/) | 2024-11-03 18:49:46 | ~3 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-241103-184618-aadc36-pr6025.zip) |
| :heavy_check_mark: | aadc36df | [#2](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6025/2/) | 2024-11-03 18:50:30 | ~4 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6025/2//consoleText) |
| :heavy_check_mark: | aadc36df | [#2](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6025/2/) | 2024-11-03 18:50:56 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-241103-184617-aadc36-pr6025.zip) |
| :heavy_check_mark: | aadc36df | [#2](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6025/2/) | 2024-11-03 18:51:25 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-241103-184613-aadc36-pr6025.zip) |
| :heavy_check_mark: | aadc36df | [#2](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6025/2/) | 2024-11-03 18:51:41 | ~5 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-241103-184611-aadc36-pr6025.aar) |
| :heavy_check_mark: | aadc36df | [#2](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6025/2/) | 2024-11-03 18:51:49 | ~5 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-241103-184612-aadc36-pr6025.zip) |
| :heavy_check_mark: | aadc36df | [#2](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6025/2/) | 2024-11-03 18:52:34 | ~6 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-241103-184617-aadc36-pr6025.zip) |
| :heavy_multiplication_x: | aadc36df | [#2](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6025/2/) | 2024-11-03 19:19:58 | ~33 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6025/2//consoleText) |
| | | | | | | |
| :heavy_multiplication_x: | 980b08fa | [#3](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6025/3/) | 2024-11-03 21:46:59 | ~1 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6025/3//consoleText) |
| :heavy_check_mark: | 980b08fa | [#3](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6025/3/) | 2024-11-03 21:48:17 | ~3 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-241103-214459-980b08-pr6025.zip) |
| :heavy_check_mark: | 980b08fa | [#3](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6025/3/) | 2024-11-03 21:49:11 | ~4 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6025/3//consoleText) |
| :heavy_check_mark: | 980b08fa | [#3](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6025/3/) | 2024-11-03 21:49:41 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-241103-214457-980b08-pr6025.zip) |
| :heavy_check_mark: | 980b08fa | [#3](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6025/3/) | 2024-11-03 21:50:06 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-241103-214454-980b08-pr6025.zip) |
| :heavy_check_mark: | 980b08fa | [#3](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6025/3/) | 2024-11-03 21:50:20 | ~5 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-241103-214453-980b08-pr6025.zip) |
| :heavy_check_mark: | 980b08fa | [#3](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6025/3/) | 2024-11-03 21:50:26 | ~5 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-241103-214452-980b08-pr6025.aar) |
| :heavy_check_mark: | 980b08fa | [#3](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6025/3/) | 2024-11-03 21:51:05 | ~6 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-241103-214457-980b08-pr6025.zip) |
| | | | | | | |
| :heavy_multiplication_x: | 0b320e32 | [#4](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6025/4/) | 2024-11-04 07:59:17 | ~1 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6025/4//consoleText) |
| :heavy_check_mark: | 0b320e32 | [#4](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6025/4/) | 2024-11-04 08:00:00 | ~2 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6025/4//consoleText) |
| :heavy_check_mark: | 0b320e32 | [#4](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6025/4/) | 2024-11-04 08:00:40 | ~2 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-241104-075740-0b320e-pr6025.zip) |
| :heavy_check_mark: | 0b320e32 | [#4](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6025/4/) | 2024-11-04 08:02:21 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-241104-075735-0b320e-pr6025.zip) |
| :heavy_check_mark: | 0b320e32 | [#4](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6025/4/) | 2024-11-04 08:02:40 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-241104-075733-0b320e-pr6025.zip) |
| :heavy_check_mark: | 0b320e32 | [#4](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6025/4/) | 2024-11-04 08:02:59 | ~5 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-241104-075731-0b320e-pr6025.aar) |
| :heavy_check_mark: | 0b320e32 | [#4](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6025/4/) | 2024-11-04 08:04:22 | ~6 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-241104-075732-0b320e-pr6025.zip) |
| :heavy_check_mark: | 0b320e32 | [#4](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6025/4/) | 2024-11-04 08:05:29 | ~7 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-241104-075735-0b320e-pr6025.zip) |
| | | | | | | |
| :heavy_multiplication_x: | c6ca8b4a | [#5](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6025/5/) | 2024-11-04 08:22:01 | ~2 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6025/5//consoleText) |
| :heavy_multiplication_x: | c6ca8b4a | [#5](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6025/5/) | 2024-11-04 08:22:09 | ~2 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6025/5//consoleText) |
| :heavy_check_mark: | c6ca8b4a | [#5](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6025/5/) | 2024-11-04 08:22:14 | ~2 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-241104-081918-c6ca8b-pr6025.zip) |
| :heavy_check_mark: | c6ca8b4a | [#5](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6025/5/) | 2024-11-04 08:23:53 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-241104-081915-c6ca8b-pr6025.zip) |
| :heavy_check_mark: | c6ca8b4a | [#5](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6025/5/) | 2024-11-04 08:24:21 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-241104-081912-c6ca8b-pr6025.zip) |
| :heavy_check_mark: | c6ca8b4a | [#5](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6025/5/) | 2024-11-04 08:24:41 | ~5 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-241104-081910-c6ca8b-pr6025.aar) |
| :heavy_check_mark: | c6ca8b4a | [#5](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6025/5/) | 2024-11-04 08:25:27 | ~6 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-241104-081911-c6ca8b-pr6025.zip) |
| :heavy_check_mark: | c6ca8b4a | [#5](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6025/5/) | 2024-11-04 08:27:07 | ~7 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-241104-081915-c6ca8b-pr6025.zip) |
| | | | | | | |
| :heavy_multiplication_x: | 0aca460b | [#6](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6025/6/) | 2024-11-04 09:15:04 | ~1 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6025/6//consoleText) |
| :heavy_check_mark: | 0aca460b | [#6](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6025/6/) | 2024-11-04 09:16:19 | ~2 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-241104-091328-0aca46-pr6025.zip) |
| :heavy_check_mark: | 0aca460b | [#6](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6025/6/) | 2024-11-04 09:17:43 | ~4 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6025/6//consoleText) |
| :heavy_check_mark: | 0aca460b | [#6](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6025/6/) | 2024-11-04 09:17:58 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-241104-091324-0aca46-pr6025.zip) |
| :heavy_check_mark: | 0aca460b | [#6](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6025/6/) | 2024-11-04 09:18:49 | ~5 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-241104-091321-0aca46-pr6025.aar) |
| :heavy_check_mark: | 0aca460b | [#6](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6025/6/) | 2024-11-04 09:18:59 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-241104-091323-0aca46-pr6025.zip) |
| :heavy_check_mark: | 0aca460b | [#6](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6025/6/) | 2024-11-04 09:19:07 | ~5 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-241104-091322-0aca46-pr6025.zip) |
| | | | | | | |
| :heavy_check_mark: | 30da93f7 | [#7](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6025/7/) | 2024-11-04 09:21:47 | ~2 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-241104-091852-30da93-pr6025.zip) |
| :heavy_check_mark: | 30da93f7 | [#7](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6025/7/) | 2024-11-04 09:23:18 | ~4 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6025/7//consoleText) |
| :heavy_check_mark: | 30da93f7 | [#7](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6025/7/) | 2024-11-04 09:23:19 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-241104-091849-30da93-pr6025.zip) |
| :heavy_check_mark: | 30da93f7 | [#7](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6025/7/) | 2024-11-04 09:24:35 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-241104-091904-30da93-pr6025.zip) |
| :heavy_check_mark: | 30da93f7 | [#7](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6025/7/) | 2024-11-04 09:24:59 | ~5 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-241104-091915-30da93-pr6025.zip) |
| :heavy_check_mark: | 30da93f7 | [#7](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6025/7/) | 2024-11-04 09:26:59 | ~8 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-241104-091852-30da93-pr6025.aar) |
| :heavy_check_mark: | 30da93f7 | [#7](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6025/7/) | 2024-11-04 09:43:56 | ~14 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-241104-092912-30da93-pr6025.zip) |
| :heavy_multiplication_x: | 30da93f7 | [#7](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6025/7/) | 2024-11-04 09:51:55 | ~33 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6025/7//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 282fcc1e | [#8](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6025/8/) | 2024-11-04 09:39:46 | ~2 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-241104-093703-282fcc-pr6025.zip) |
| :heavy_check_mark: | 282fcc1e | [#8](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6025/8/) | 2024-11-04 09:41:29 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-241104-093659-282fcc-pr6025.zip) |
| :heavy_check_mark: | 282fcc1e | [#8](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6025/8/) | 2024-11-04 09:41:33 | ~4 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6025/8//consoleText) |
| :heavy_check_mark: | 282fcc1e | [#8](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6025/8/) | 2024-11-04 09:42:14 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-241104-093658-282fcc-pr6025.zip) |
| :heavy_check_mark: | 282fcc1e | [#8](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6025/8/) | 2024-11-04 09:42:52 | ~5 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-241104-093657-282fcc-pr6025.zip) |
| :heavy_check_mark: | 282fcc1e | [#8](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6025/8/) | 2024-11-04 09:45:22 | ~8 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-241104-093656-282fcc-pr6025.aar) |
| :heavy_check_mark: | 282fcc1e | [#8](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6025/8/) | 2024-11-04 09:55:45 | ~11 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-241104-094405-282fcc-pr6025.zip) |
| :heavy_multiplication_x: | 282fcc1e | [#8](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6025/8/) | 2024-11-04 10:26:52 | ~34 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6025/8//consoleText) |
| | | | | | | |
| :heavy_multiplication_x: | c42b96b6 | [#9](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6025/9/) | 2024-11-21 13:27:22 | ~2 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6025/9//consoleText) |
| :heavy_multiplication_x: | c42b96b6 | [#9](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6025/9/) | 2024-11-21 13:27:56 | ~3 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6025/9//consoleText) |
| :heavy_check_mark: | c42b96b6 | [#9](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6025/9/) | 2024-11-21 13:28:46 | ~4 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-241121-132431-c42b96-pr6025.zip) |
| :heavy_check_mark: | c42b96b6 | [#9](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6025/9/) | 2024-11-21 13:29:01 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-241121-132426-c42b96-pr6025.zip) |
| :heavy_check_mark: | c42b96b6 | [#9](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6025/9/) | 2024-11-21 13:29:15 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-241121-132423-c42b96-pr6025.zip) |
| :heavy_check_mark: | c42b96b6 | [#9](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6025/9/) | 2024-11-21 13:30:03 | ~5 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-241121-132421-c42b96-pr6025.aar) |
| :heavy_check_mark: | c42b96b6 | [#9](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6025/9/) | 2024-11-21 13:30:16 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-241121-132425-c42b96-pr6025.zip) |
| :heavy_check_mark: | c42b96b6 | [#9](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6025/9/) | 2024-11-21 13:31:31 | ~7 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-241121-132426-c42b96-pr6025.zip) |
| | | | | | | |
| :heavy_multiplication_x: | 2d03ecc4 | [#10](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6025/10/) | 2024-11-21 14:14:31 | ~2 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6025/10//consoleText) |
| :heavy_check_mark: | 2d03ecc4 | [#10](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6025/10/) | 2024-11-21 14:15:05 | ~3 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-241121-141152-2d03ec-pr6025.zip) |
| :heavy_multiplication_x: | 2d03ecc4 | [#10](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6025/10/) | 2024-11-21 14:15:19 | ~3 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6025/10//consoleText) |
| :heavy_check_mark: | 2d03ecc4 | [#10](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6025/10/) | 2024-11-21 14:16:19 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-241121-141146-2d03ec-pr6025.zip) |
| :heavy_check_mark: | 2d03ecc4 | [#10](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6025/10/) | 2024-11-21 14:16:43 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-241121-141145-2d03ec-pr6025.zip) |
| :heavy_check_mark: | 2d03ecc4 | [#10](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6025/10/) | 2024-11-21 14:16:52 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-241121-141146-2d03ec-pr6025.zip) |
| :heavy_check_mark: | 2d03ecc4 | [#10](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6025/10/) | 2024-11-21 14:17:01 | ~5 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-241121-141143-2d03ec-pr6025.aar) |
| :heavy_check_mark: | 2d03ecc4 | [#10](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6025/10/) | 2024-11-21 14:19:36 | ~7 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-241121-141148-2d03ec-pr6025.zip) |
| | | | | | | |
| :heavy_check_mark: | 02e3ad59 | [#11](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6025/11/) | 2024-11-21 14:25:53 | ~2 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-241121-142259-02e3ad-pr6025.zip) |
| :heavy_check_mark: | 02e3ad59 | [#11](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6025/11/) | 2024-11-21 14:27:31 | ~4 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-241121-142252-02e3ad-pr6025.zip) |
| :heavy_check_mark: | 02e3ad59 | [#11](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6025/11/) | 2024-11-21 14:27:55 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-241121-142256-02e3ad-pr6025.zip) |
| :heavy_multiplication_x: | 02e3ad59 | [#11](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6025/11/) | 2024-11-21 14:28:00 | ~5 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6025/11//consoleText) |
| :heavy_check_mark: | 02e3ad59 | [#11](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6025/11/) | 2024-11-21 14:28:00 | ~5 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-241121-142252-02e3ad-pr6025.zip) |
| :heavy_check_mark: | 02e3ad59 | [#11](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6025/11/) | 2024-11-21 14:30:14 | ~7 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-241121-142256-02e3ad-pr6025.zip) |
| :heavy_check_mark: | 02e3ad59 | [#11](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6025/11/) | 2024-11-21 14:30:15 | ~7 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-241121-142251-02e3ad-pr6025.aar) |
| :heavy_multiplication_x: | 02e3ad59 | [#11](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6025/11/) | 2024-11-21 14:56:58 | ~33 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6025/11//consoleText) |
| | | | | | | |
| :heavy_check_mark: | bbe71914 | [#12](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6025/12/) | 2024-11-21 15:44:39 | ~2 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-241121-154150-bbe719-pr6025.zip) |
| :heavy_check_mark: | bbe71914 | [#12](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6025/12/) | 2024-11-21 15:46:25 | ~4 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-241121-154143-bbe719-pr6025.zip) |
| :heavy_check_mark: | bbe71914 | [#12](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6025/12/) | 2024-11-21 15:46:33 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-241121-154146-bbe719-pr6025.zip) |
| :heavy_check_mark: | bbe71914 | [#12](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6025/12/) | 2024-11-21 15:46:43 | ~5 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-241121-154142-bbe719-pr6025.zip) |
| :heavy_multiplication_x: | bbe71914 | [#12](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6025/12/) | 2024-11-21 15:46:48 | ~4 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6025/12//consoleText) |
| :heavy_check_mark: | bbe71914 | [#12](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6025/12/) | 2024-11-21 15:49:56 | ~8 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-241121-154141-bbe719-pr6025.aar) |
| :heavy_check_mark: | bbe71914 | [#12](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6025/12/) | 2024-11-21 15:50:00 | ~8 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-241121-154147-bbe719-pr6025.zip) |
| :heavy_check_mark: | bbe71914 | [#12](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6025/12/) | 2024-11-21 16:15:40 | ~33 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6025/12//consoleText) |
</details>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :x: | 0ae52530 | [#13](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6025/13/) | 2024-11-21 17:27:18 | ~33 sec | `macos` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6025/13/consoleText) |
| :heavy_check_mark: | 0ae52530 | [#13](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6025/13/) | 2024-11-21 17:29:41 | ~2 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-241121-172648-0ae525-pr6025.zip) |
| :heavy_check_mark: | 0ae52530 | [#13](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6025/13/) | 2024-11-21 17:30:56 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-241121-172642-0ae525-pr6025.zip) |
| :heavy_check_mark: | 0ae52530 | [#13](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6025/13/) | 2024-11-21 17:31:23 | ~4 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6025/13//consoleText) |
| :heavy_check_mark: | 0ae52530 | [#13](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6025/13/) | 2024-11-21 17:32:53 | ~6 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-241121-172643-0ae525-pr6025.zip) |
| :heavy_check_mark: | 0ae52530 | [#13](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6025/13/) | 2024-11-21 17:33:21 | ~6 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-241121-172641-0ae525-pr6025.aar) |
| :heavy_check_mark: | 0ae52530 | [#13](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6025/13/) | 2024-11-21 17:34:49 | ~8 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-241121-172645-0ae525-pr6025.zip) |
| :heavy_check_mark: | 0ae52530 | [#14](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6025/14/) | 2024-11-21 17:46:04 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-241121-174228-0ae525-pr6025.zip) |
| :heavy_check_mark: | 0ae52530 | [#13](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6025/13/) | 2024-11-21 17:58:59 | ~32 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6025/13//consoleText) |
| :heavy_check_mark: | 0ae52530 | [#14](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6025/14/) | 2024-11-22 07:49:45 | ~3 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-241122-074642-0ae525-pr6025.zip) |
| :heavy_check_mark: | 0ae52530 | [#14](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6025/14/) | 2024-11-22 07:51:19 | ~4 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6025/14//consoleText) |
| :heavy_check_mark: | 0ae52530 | [#15](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6025/15/) | 2024-11-22 07:51:54 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-241122-074638-0ae525-pr6025.zip) |
| :heavy_check_mark: | 0ae52530 | [#14](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6025/14/) | 2024-11-22 07:52:24 | ~5 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-241122-074636-0ae525-pr6025.zip) |
| :heavy_check_mark: | 0ae52530 | [#14](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6025/14/) | 2024-11-22 07:52:24 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-241122-074637-0ae525-pr6025.zip) |
| :heavy_check_mark: | 0ae52530 | [#14](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6025/14/) | 2024-11-22 07:52:30 | ~5 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-241122-074634-0ae525-pr6025.aar) |
| :heavy_check_mark: | 0ae52530 | [#14](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6025/14/) | 2024-11-22 07:54:13 | ~7 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-241122-074638-0ae525-pr6025.zip) |
| :heavy_check_mark: | 0ae52530 | [#14](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6025/14/) | 2024-11-22 08:19:01 | ~32 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6025/14//consoleText) |
| | | | | | | |
| :heavy_check_mark: | e17feaf2 | [#15](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6025/15/) | 2024-11-22 07:52:48 | ~2 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-241122-074948-e17fea-pr6025.zip) |
| :heavy_check_mark: | e17feaf2 | [#16](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6025/16/) | 2024-11-22 07:55:49 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-241122-075158-e17fea-pr6025.zip) |
| :heavy_check_mark: | e17feaf2 | [#15](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6025/15/) | 2024-11-22 07:57:10 | ~5 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6025/15//consoleText) |
| :heavy_check_mark: | e17feaf2 | [#15](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6025/15/) | 2024-11-22 07:58:52 | ~6 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-241122-075229-e17fea-pr6025.zip) |
| :heavy_check_mark: | e17feaf2 | [#15](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6025/15/) | 2024-11-22 08:02:22 | ~9 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-241122-075228-e17fea-pr6025.zip) |
| :heavy_check_mark: | e17feaf2 | [#15](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6025/15/) | 2024-11-22 08:02:32 | ~8 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-241122-075420-e17fea-pr6025.zip) |
| :heavy_check_mark: | e17feaf2 | [#15](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6025/15/) | 2024-11-22 08:08:53 | ~16 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-241122-075235-e17fea-pr6025.aar) |
| :heavy_check_mark: | e17feaf2 | [#15](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6025/15/) | 2024-11-22 08:52:31 | ~33 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6025/15//consoleText) |

osmaczko committed:
chore(logging)_: introduce geth adapter

osmaczko committed:
chore(logging)_: switch to zap.Logger as central logger

Set zap.Logger as the  primary logger for status-go. All geth logs are
now proxied through zap.Logger.

closes: #6029

igor-sirotin commented on logutils/geth_adapter.go:
```diff
@@ -0,0 +1,45 @@
+func lvlFromString(lvlString string) (zapcore.Level, error) {
+	switch strings.ToLower(lvlString) {
+	case "trace", "trce":
+		return zapcore.DebugLevel, nil // zap does not have a trace level, using DebugLevel as closest
```
I think we should skip `trace` messages. It will pollute debug logs too much.

igor-sirotin commented on logutils/geth_adapter_test.go:
```diff
@@ -0,0 +1,44 @@
+
+	buffer.Reset()
+	log.Info("should be printed")
+	require.Regexp(t, `INFO\s+'INFO\s*\[.*\]\s*should be printed '`, buffer.String())
```
Why is `INFO` duplicated in the log message? :think

osmaczko replied:
To keep it simple geth logs are wrapped as is (as string): https://github.com/status-im/status-go/blob/c42b96b6d3c185665b167c1f2133a56c0eb73888/logutils/geth_adapter.go#L21-L22

Good example is:
https://github.com/status-im/status-go/blob/c42b96b6d3c185665b167c1f2133a56c0eb73888/logutils/geth_adapter_test.go#L42-L43

igor-sirotin commented on cmd/statusd/main.go:
```diff
@@ -46,7 +45,6 @@ const (
 var (
 	configFiles                    configFlags
 	logLevel                       = flag.String("log", "", `Log level, one of: "ERROR", "WARN", "INFO", "DEBUG", and "TRACE"`)
-	logWithoutColors               = flag.Bool("log-without-color", false, "Disables log colors")
```
Probably better to keep the flag, but deprecate it. I'm not sure of all usages of our `cmd`s. 
Or ensure that it's not used anywhere. And mark the commit as a `breaking change`

qfrank replied:
Pls don't remove it,I like the colorful output. Especially, we will frequently use the status backend server now.
<img width="1499" alt="image" src="https://github.com/user-attachments/assets/6f6f2a3c-7c88-490f-9449-99674217dc3f">

osmaczko replied:
I kept the flag and the logic but colorized output is a bit tricky with zap.logger. Will create a followup issue to address that.

osmaczko replied:
https://github.com/status-im/status-go/issues/6116

igor-sirotin commented on logutils/logger.go:
```diff
@@ -1,27 +1,52 @@
+
+func defaultEncoder() zapcore.Encoder {
+	encoderConfig := zap.NewDevelopmentEncoderConfig()
+	encoderConfig.EncodeTime = utcTimeEncoder(encoderConfig.EncodeTime)
```
👍

igor-sirotin commented on logutils/override.go:
```diff
@@ -19,82 +19,55 @@ type LogSettings struct {
-	if fileOpts.Filename != "" {
-		if fileOpts.MaxBackups == 0 {
+
+	if settings.File != "" {
```
Can you please invrert the condition, so that the shorter branch goes first?

qfrank commented on cmd/ping-community/main.go:
```diff
@@ -73,8 +71,11 @@ func init() {
 // nolint:gocyclo
 func main() {
-	colors := terminal.IsTerminal(int(os.Stdin.Fd()))
-	if err := logutils.OverrideRootLog(true, "ERROR", logutils.FileOptions{}, colors); err != nil {
```
why color support removed?

osmaczko replied:
https://github.com/status-im/status-go/pull/6025#discussion_r1852117868

qfrank commented on logutils/core.go:
```diff
@@ -53,6 +53,14 @@ func (core *Core) Enabled(lvl zapcore.Level) bool {
 	return core.level.Enabled(lvl)
 }
 
+func (core *Core) Level() zapcore.Level {
```
can we rename the file to zap_core.go?

qfrank commented on logutils/logger_test.go:
```diff
@@ -1,18 +0,0 @@
-	handler := log.StreamHandler(buf, log.TerminalFormat(false))
-	require.NoError(t, enableRootLog("debug", handler))
-	log.Debug("hello")
-	require.Contains(t, buf.String(), "logutils/logger_test.go:16")
```
will it contain the origin info after replaced with zap?

osmaczko replied:
Yes, I brought back the test just to be sure.

qfrank commented on logutils/override.go:
```diff
@@ -19,82 +19,55 @@ type LogSettings struct {
-func OverrideRootLog(enabled bool, levelStr string, fileOpts FileOptions, terminal bool) error {
-	if !enabled {
-		disableRootLog()
+	if settings.MobileSystem {
```
it seems we never set MobileSystem to true in frontend, do you know why we need this ?  🤔

osmaczko replied:
I have no idea, was introduced there 5bce7e793f835f5c14dcbe63d75ee992f7d81f04. If you are sure it is not used, then I can remove it altogether.
